README.md

```diff
@@ -44,7 +44,7 @@ Please note that both the WebGPU and WGSL spec are still under development.
 2. Add the following snippet to your config file:
 
    ```lua
-   vim.cmd[[au BufRead,BufNewFile *.wgsl	set filetype=wgsl]]
+   vim.filetype.add({extension = {wgsl = "wgsl}})
```

Comment:
```suggestion
   vim.filetype.add({extension = {wgsl = "wgsl"}})
```

Reply:
Or, why to never make changes without syntax highlighting on :D

Thanks for spotting it, I've amended the commit.

Reply:
Thank you. I'm sorry for the initial delay.